governance/permissionless-proposals.md

```diff
@@ -0,0 +1,92 @@
+2. **Proposal Finalization**: After the proposal has been drafted and shared on the forum, delegates provide feedback, ask clarifying questions, and propose changes. Once the delegate incorporates this input and the proposal is refined, the delegate updates its status to "Final Draft". At this point, the delegate must create an on-chain transaction to create the proposal metadata record in the `ProposalValidator` by calling `submitProposal()`, which can be done either through Etherscan or Agora UI. This call emits a `ProposalSubmitted` event, which includes the `proposalId` and other relevant data. Agora listens for this event and uses it to fetch and display the finalized proposal on its UI, allowing delegates to begin the sign-off process.
+    
+```mermaid
+flowchart LR
+    Delegate([Delegate]) -->|Submit DRAFT proposal| OPGov[OPGov Forum]
+    Community([Community]) -->|Provide feedback| OPGov
+    OPGov -->|submitProposal once FINAL| OnChainGov[ProposalValidator]
+```
+    
+3. **On-chain Sign-Offs via Agora**: With the final proposal ready, the top 100 delegates can start signing-off on the on-chain proposal using the `proposalId` through Agora. Their sign-offs are recorded via the `approveProposal(...)` function in the `ProposalValidator` contract, that emits a `ProposalApproved` event. Agora will display the real-time signature status for each proposal.
+4. **Move to vote via** `ProposalValidator`**:** Once the required number of sign-offs is reached, the "Approve Proposal" button in Agora changes to "Propose". Any user can click this button to call `moveToVote(...)` on the `ProposalValidator`. If the proposal meets the necessary rules, such as having the required top 100 delegates sign-offs, being submitted within the correct time window, not going over the distribution rate limit, and adhering to any proposal type-specific threshold it will be moved to the Governor for voting. This is done by internally calling `proposeWithModule(...)` **within `moveToVote(...)`.
+5. **Governor:** If the `ProposalValidator` checks that the proposal is valid, it forwards the proposal to the OP Governor contract. Then, the voting time begins as usual.
+
+```mermaid
+flowchart LR
+    subgraph Agora UI
+        AgoraUI -->|listens to ProposalSubmitted| A2[Agora UI]
+    end
+
+    A2 -->|Displays new proposal<br>based on Proposal Type| B[Agora UI]
+    C[Top Delegates] -->|approve proposal Id| B
+    B -->|approveProposal| D[ProposalValidator]
+
+    E[Approved Proposer] -->|moveToVote| F[Agora UI]
+    F -->|moveToVote| G[ProposalValidator]
+    G -->|propose| H[OPGov]
+```
+
+Detailed flows for each Proposal Type can be seen [here](https://www.figma.com/board/V0zh1Lg5QqhGfhVrJaIJVu/Permissionless-Proposals-DRAFT?node-id=0-1&p=f&t=sFuVhRioJR1kAIzV-0).
+
+# Alternatives Considered
```

Comment:
Please fix language to match the OPerating Manual in the sections below:
"sign-off" -> "approve"
"Final Draft" -> "Final"
"propose(...)" -> "moveToVote(...)"